src/lib.rs

```diff
@@ -348,6 +348,18 @@ impl Span {
         Span::_new(imp::Span::call_site())
     }
 
+    /// The span located at the invocation of the procedural macro, but with
+    /// local variables, labels, and `$crate` resolved at the definition site
+    /// of the macro.
+    ///
+    /// `macro_rules` behaves like this in terms of hygiene.
```

Comment:
This is a rephrasing of the [proc_macro::Span::mixed_site() documentation](https://github.com/rust-lang/rust/blob/09739c22db948a7a2ed13e8f466cb22adb1d7818/src/libproc_macro/lib.rs#L309-L312), but I was unable to get `$crate` to work in a proc macro -- it just errors with "expected identifier, found `$`". Do you know if that is expected to work, or is that referring only to `macro_rules`?